lib/RTCSession.js

```diff
@@ -266,11 +266,14 @@ module.exports = class RTCSession extends EventEmitter
     debug('connect()');
 
     const originalTarget = target;
-    const eventHandlers = options.eventHandlers || {};
+    const eventHandlers = Utils.cloneObject(options.eventHandlers);
     const extraHeaders = Utils.cloneArray(options.extraHeaders);
-    const mediaConstraints = options.mediaConstraints || { audio: true, video: true };
+    const mediaConstraints = Utils.cloneObject(options.mediaConstraints, {
+      audio : true,
+      video : false
+    });
```

Comment:
this is wrong, default value for `video` is true.

Reply:
fixed thanks